frontend/src/helper/board/blurFilter.ts

```diff
@@ -6,6 +7,12 @@ export function cardBlur(hideCards: boolean, card: CardType, userId: string) {
 	return hideCards && card.createdBy?._id !== userId ? 'blur($sizes$6)' : 'none';
 }
 
+export function cardFooterBlur(hideCards: boolean, createdBy: any, userId: string) {
+	return hideCards && (createdBy?._id !== userId || createdBy?._id === undefined)
```

Comment:
```suggestion
export function cardFooterBlur(hideCards: boolean, createdByUserId?: string, userId: string) {
	return hideCards && (createdByUserId !== userId || createdByUserId === undefined)
```

Just a suggestion to avoid `any` types